packages/ui/origin-ui-core/src/components/organization/Organization.tsx

```diff
@@ -7,15 +7,16 @@ import { useLinks } from '../../hooks';
 export const Organization = () => {
     const { organizationPageUrl } = useLinks();
     const organizationMenuList = useOrganizationMenu();
-    const firstNotHiddenRoute = organizationMenuList.filter((i) => i.show)[0]?.key;
+    const displayableMenuList = organizationMenuList.filter((i) => i.show);
+    const firstNotHiddenRoute = displayableMenuList[0]?.key;
 
     return (
         <div className="PageWrapper">
             <Route
                 path={`${organizationPageUrl}/:key/:id?`}
                 render={(props) => {
                     const key = props.match.params.key;
-                    const matches = organizationMenuList.filter((item) => {
+                    const matches = displayableMenuList.filter((item) => {
```

Comment:
Here the match should work only with displayable items, not with all available (IMO)